diffs/20241114-Ethereum-0xC13e21B648A5Ee794902342038FF3aDAB66BE987-pre_20241114-Ethereum-0xC13e21B648A5Ee794902342038FF3aDAB66BE987-post.md

```diff
@@ -0,0 +1,88 @@
+## Reserve changes
+
+### Reserve altered
+
+#### WBTC ([0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599](https://etherscan.io/address/0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599))
+
+| description | value before | value after |
+| --- | --- | --- |
+| liquidationThreshold | 70 % | 65 % |
+
+
+#### wstETH ([0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0](https://etherscan.io/address/0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0))
+
+| description | value before | value after |
+| --- | --- | --- |
+| borrowCapGap | 100 | 5,000 |
+| maxBorrowCap | 3,000 | 100,000 |
+
+
+#### WETH ([0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2](https://etherscan.io/address/0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2))
+
+| description | value before | value after |
+| --- | --- | --- |
+| interestRateStrategy | [0x6fd32465a23aa0DBaE0D813B7157D8CB2b08Dae4](https://etherscan.io/address/0x6fd32465a23aa0DBaE0D813B7157D8CB2b08Dae4) | [0xf4268AeC16d13446381F8a2c9bB05239323756ca](https://etherscan.io/address/0xf4268AeC16d13446381F8a2c9bB05239323756ca) |
+| variableRateSlope1 | 2.8991774151952311 % | 2.3991774151952311 % |
+| baseStableBorrowRate | 2.8991774151952311 % | 2.3991774151952311 % |
```

Comment:
fyi: having local changes when running `forge test` 
<img width="1028" alt="image" src="https://github.com/user-attachments/assets/ad564725-adf4-4ff7-af40-6db9387cd4dd">

Reply:
We stopped using those graphs. I think you have the wrong version of the CLI tool.

Reply:
feel free to close 👍